packages/base/src/util/ColorConversion.js

```diff
@@ -367,4 +389,6 @@ export {
 	HSLToRGB,
 	HEXToRGB,
 	RGBToHSL,
+	RGBToRGB,
```

Comment:
can be renamed to RGBStringToRGB, otherwise it makes no sense.

Reply:
done